libs/wallet/src/web3-react/hooks/useSwitchNetwork.ts

```diff
@@ -0,0 +1,15 @@
+import { useWeb3React } from '@web3-react/core'
+import { useCallback } from 'react'
+import { switchChain } from '@cowprotocol/wallet'
```

Comment:
this import should be imported using relative (since its in the same library)?

Reply:
Fixed